structs.go

```diff
@@ -644,7 +653,7 @@ type Role struct {
 	// The permissions of the role on the guild (doesn't include channel overrides).
 	// This is a combination of bit masks; the presence of a certain permission can
 	// be checked by performing a bitwise AND between this int and the permission.
-	Permissions string `json:"permissions"`
+	Permissions int `json:",string,permissions"`
```

Comment:
Sorry, I should've specified more clearly—the `,string` option is appended to the value in the quotes, i.e. `json:"permissions,string"`.